### PR TITLE
Update Oxygen for RC2 (WTP RC3)

### PR DESCRIPTION
**Pull request by briandealwis**
Using Oxygen RC2 (…/releases/oxygen/XXX) as RC3 is being staged as we speak at a transient URL.
But WTP RC3 is at a stable URL.  There are no changes to the Platform from RC2 &rarr; RC3 that I can think of that will affect WTP RC2.

### CODE REVIEW

File: eclipse/oxygen/gcp-eclipse-oxygen.tpd [hunk 1]
```diff
@@ -12,10 +12,11 @@
 target "GCP for Eclipse Oxygen" with source requirements
 
 location "http://download.eclipse.org/cbi/updates/license" {
-    org.eclipse.license.feature.group    
+    org.eclipse.license.feature.group
 }
 
-location "http://download.eclipse.org/releases/oxygen/201702031000/" {
+// 201705191000 = Oxygen RC2
```
Comment (chanseokoh) [inline eclipse/oxygen/gcp-eclipse-oxygen.tpd]: You mean RC3?

Comment (chanseokoh) [inline eclipse/oxygen/gcp-eclipse-oxygen.tpd]: Never mind.

**Comment by elharo** [comment]
[INFO] ------------------------------------------------------------------------

[INFO] Skipping jacoco

[INFO] This project has been banned from the build due to previous failures.

**Comment by elharo** [comment]
This failure may be flaky

```
Tests run: 9, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 84.646 sec <<< FAILURE! - in com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest

testMostImportantFile(com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest)  Time elapsed: 31.902 sec

testAppEngineLibrariesAdded(com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest)  Time elapsed: 17.613 sec

testNoTestClassesInDeploymentAssembly(com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest)  Time elapsed: 7.521 sec  <<< FAILURE!

java.lang.AssertionError: null

	at org.junit.Assert.fail(Assert.java:86)

	at org.junit.Assert.assertTrue(Assert.java:41)

	at org.junit.Assert.assertFalse(Assert.java:64)

	at org.junit.Assert.assertFalse(Assert.java:74)

	at com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest.assertNoTestClassesInDeploymentAssembly(CreateAppEngineStandardWtpProjectTest.java:212)

	at com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest.testNoTestClassesInDeploymentAssembly(CreateAppEngineStandardWtpProjectTest.java:201)
```

**Pull request merged**